codechecker_common/plist_parser.py

```diff
@@ -301,6 +303,9 @@ def remove_report_from_plist(plist_file_obj, skip_handler):
     report_data = None
     try:
         report_data = parse_plist(plist_file_obj)
+    except plistlib.InvalidFileException:
```

Comment:
This exception will come from the `parse_plist` function from the following line:
https://github.com/Ericsson/codechecker/blob/afc73ed9ffba37286a08a28079861fc41943ea08/codechecker_common/plist_parser.py#L111

So instead of handling this exception in two places we can handle it on one place.

Isn't it necessary to handle this exception every places where we load a plist file and this kind of exception can happend?
https://github.com/Ericsson/codechecker/blob/afc73ed9ffba37286a08a28079861fc41943ea08/tools/codechecker_report_hash/codechecker_report_hash/hash.py#L301
https://github.com/Ericsson/codechecker/blob/afc73ed9ffba37286a08a28079861fc41943ea08/tools/plist_to_html/plist_to_html/PlistToHtml.py#L371